neo/sys/posix/platform_linux.cpp

```diff
@@ -56,21 +56,32 @@ Sys_EXEPath
 */
 const char* Sys_EXEPath()
 {
-	static char	buf[ 1024 ];
```

Comment:
Don't get me wrong, I don't think that always just using PATH_MAX is the right thing to do and "proper" solutions are purely academical.
If you write a program that's supposed to handle arbitrary files (an `ls` clone, an image viewer, a text editor, ...) you should really try to do these things right (I wonder how many tools, even basic ones like `ls`, `du`, `less` etc really handle this 100% correctly). But for a game that only handles files in its installation directory and in `$HOME/.foo`, assuming a reasonable max path length should be acceptable.